crates/rpc-types-txpool/src/txpool.rs

```diff
@@ -204,7 +204,6 @@ mod tests {
         "blockNumber": null,
         "from": "0x00000000863b56a3c1f0f1be8bc4f8b7bd78f57a",
         "gas": "0x2af9e",
-        "gasPrice": "0x218711a00",
```

Comment:
need someone to confirm if this is correct. I believe these can be deleted as returning this field for eip1559 txns is deprecated

Reply:
yep, it should be fine to remove non-legacy